server/lib/fusor/engine.rb

```diff
@@ -8,6 +8,18 @@ class Engine < ::Rails::Engine
     config.autoload_paths += Dir["#{config.root}/app/overrides"]
     config.autoload_paths += Dir["#{config.root}/app/serializers"]
 
+    initializer 'fusor.silenced_logger', :after => :build_middleware_stack do |app|
+      # Add additional paths below if you want logging silenced
+      #  we want the polling of ForemanTasksController#show silenced to reduce noise in logs
+      silenced_paths = ["api/v21/foreman_tasks/"]
+
+      Rails.logger.warn "fusor_server will update 'Katello.config.logging.ignored_paths' to silence logging of paths '#{silenced_paths}'"
+      for sil_path in silenced_paths
+        Katello.config.logging.ignored_paths.push(sil_path)
```

Comment:
@bbuckingham do you know of a way I can ensure that the 'fusor.silenced_logger' initializer is run after Katello has run it's "silenced_logger" initializer? 

Looking at the error from travis it seems like our initializer for fusor_server is running first.  During testing Katello happened to load first so I didn't hit any issues.

Any thoughts?